resource-managers/kubernetes/core/src/main/scala/org/apache/spark/deploy/k8s/KubernetesUtils.scala

```diff
@@ -89,26 +56,16 @@ private[spark] object KubernetesUtils {
   def getOnlyRemoteFiles(uris: Iterable[String]): Iterable[String] = {
     uris.filter { uri =>
       val scheme = Utils.resolveURI(uri).getScheme
-      scheme != "file" && scheme != "local"
+      scheme != "local"
     }
   }
 
-  private def resolveFileUri(
-      uri: String,
-      fileDownloadPath: String,
-      assumesDownloaded: Boolean): String = {
+  private def resolveFileUri(uri: String): String = {
     val fileUri = Utils.resolveURI(uri)
-    val fileScheme = Option(fileUri.getScheme).getOrElse("file")
+    val fileScheme = Option(fileUri.getScheme).getOrElse("non-local")
```

Comment:
Don't think we should be putting an arbitrary scheme here?

Reply:
Valid, `match` statement is on `_` so it didn't matter what I put in. I can change this if need be

Reply:
Should just stick with `file` I think.

Reply:
Done.